RELEASE_NOTES.rst

```diff
@@ -21,6 +21,271 @@
+  and fixes issues where SLAs stop the system from seeing changes to existing dag files. Additional metrics added to help track queue state.
+
+The ``cleanup()`` method in BaseTrigger is now defined as asynchronous (following async/await) pattern (#30152).
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+  This is potentially a breaking change for any custom trigger implementations that override the ``cleanup()``
+  method and uses synchronous code, however using synchronous operations in cleanup was technically wrong,
+  because the method was executed in the main loop of the Triggerer and it was introducing unnecessary delays
+  impacting other triggers. The change is unlikely to affect any existing trigger implementations.
+
+The gauge ``scheduler.tasks.running`` no longer exist (#30374)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+  The gauge has never been working and its value has always been 0. Having an accurate
+  value for this metric is complex so it has been decided that removing this gauge makes
+  more sense than fixing it with no certainty of the correctness of its value.
+
+Consolidate handling of tasks stuck in queued under new ``task_queued_timeout`` config (#30375)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+  Logic for handling tasks stuck in the queued state has been consolidated, and the all configurations
+  responsible for timing out stuck queued tasks have been deprecated and merged into
+  ``[scheduler] task_queued_timeout``. The configurations that have been deprecated are
+  ``[kubernetes] worker_pods_pending_timeout``, ``[celery] stalled_task_timeout``, and
+  ``[celery] task_adoption_timeout``. If any of these configurations are set, the longest timeout will be
+  respected. For example, if ``[celery] stalled_task_timeout`` is 1200, and ``[scheduler] task_queued_timeout``
+  is 600, Airflow will set ``[scheduler] task_queued_timeout`` to 1200.
+
+Improvement Changes
+^^^^^^^^^^^^^^^^^^^
+
+Display only the running configuration in configurations view (#28892)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+  The configurations view now only displays the running configuration. Previously, the default configuration
+  was displayed at the top but it was not obvious whether this default configuration was overridden or not.
+  Subsequently, the non-documented endpoint ``/configuration?raw=true`` is deprecated and will be removed in
+  Airflow 3.0. The HTTP response now returns an additional ``Deprecation`` header. The ``/config`` endpoint on
+  the REST API is the standard way to fetch Airflow configuration programmatically.
+
+Explicit skipped states list for ExternalTaskSensor (#29933)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+  ExternalTaskSensor now has an explicit ``skipped_states`` list
```

Comment:
```suggestion
ExternalTaskSensor now has an explicit ``skipped_states`` list
```

Reply:
I feel like this can just be folded into the main list of improvements.

Reply:
The title is already on the list but this seems different since it was from fragment

Reply:
Ah, then I don't see much value in having this extra chunk then. Just in the list is enough for this change imo.